docker-compose/ups-release/helper/hack.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-export KEYCLOAK_PORT_8080_TCP_ADDR=$(ping -c 1 keycloak | grep -Eo -m 1 '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}';)
-echo $KEYCLOAK_PORT_8080_TCP_ADDR
+export KEYCLOAK_SERVICE_HOST=$(ping -c 1 keycloak | grep -Eo -m 1 '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}';)
+echo KEYCLOAK runs on: $KEYCLOAK_SERVICE_HOST
```

Comment:
Not actually about the code, but maybe a more descriptive filename is in order.

Reply:
:-)